benchmark/aggregator-pom.xml

```diff
@@ -0,0 +1,13 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>fr.vidal.oss</groupId>
+    <artifactId>atom-jaxb-aggregator</artifactId>
+    <version>0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    
+    <modules>
```

Comment:
I've done that because I don't want to move `atom-jaxb` in a Maven module, but... 🤷‍♂️